frontend/amundsen_application/static/js/features/ColumnList/ColumnType/parser.ts

```diff
@@ -91,8 +97,15 @@ function parseNestedTypeHelper(
           nextStartIndex++;
         }
 
+        const nestedString = columnType.substring(
+          startIndex,
+          nextStartIndex - 1
+        );
+        const spaceIndex = nestedString.indexOf(' ');
         children.push({
           head: columnType.substring(startIndex, currentIndex + 1),
+          name: (spaceIndex !== -1) ? nestedString.substring(0, spaceIndex) : '',
+          col_type: nestedString.substring(spaceIndex + 1),
```

Comment:
Got a bug where sometimes the last nested column is excluded from the col_type string. Need to check this substring.